styles.css

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Verdana, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid black;
+  background-color: green;
+  font-family: monospace;
```

Comment:
if possible don´t mix font family within one project if it´s not necessary
rest:  LGTM👌